packages/arb-token-bridge-ui/src/components/TransactionHistory/TransactionsTable/TransactionsTableWithdrawalRow.tsx

```diff
@@ -369,34 +374,56 @@ export function TransactionsTableWithdrawalRow({
     [l2.network, tx.tokenAddress, tx.asset]
   )
 
+  const customAddressTxPadding = useMemo(
+    () => (isCustomDestinationAddressTx(tx) ? 'pb-11' : ''),
+    [tx]
+  )
+
+  if (!tx.sender || !address) {
```

Comment:
Can this happen? I guess if we fall back to event logs?

Reply:
Yeah, event logs don't give us the sender

Reply:
So we don't render the txns from event logs?